dist/index.html

```diff
@@ -3,17 +3,86 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- font awesome file -->
+    <link rel="stylesheet" href="./assets/css/font-awesome.min.css" />
+
     <link rel="stylesheet" href="./assets/css/main.css" />
     <title>Charity</title>
   </head>
   <body>
     <header>
-      <section class="contact">
-        Welcome text and social media Links
+      <section class="contact bg-c100 text-c700 py-2">
+        <div class="container flex justify-between">
+          <div class="welcome-text text-sm">
+            Welcome to the best
+            <span class="text-c300 underline italic">Lovims</span> charity
+            platform
+          </div>
+          <div class="social flex text-sm">
+            <div>Follow us:</div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-facebook-square" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-pinterest-p" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-instagram" aria-hidden="true"></i>
+              </a>
+            </div>
+          </div>
+        </div>
       </section>
 
-      <section>
-        Logo And Contact Info
+      <section class="logo-section py-8">
+        <div class="container py-2 flex">
+          <div class="logo w-1/2">
+            <img src="assets/Logo@1X.png" alt="" />
+          </div>
+
+          <div class="contact-info flex justify-between w-full">
+            <div class="email flex border-r border-c700 pr-8">
+              <div class="icon items-center text-c500">
+                <i class="fa fa-envelope-o fa-2x py-2 pr-4"></i>
```

Comment:
Great job so far by using Font-awesome 👍 
but we need to use same images of the PSD.
I have attached the three images( email, phone, location icons) so that you can user 
![email-icon](https://user-images.githubusercontent.com/8650435/79673624-6978a280-81ec-11ea-92df-344b08369ac2.png)
![phone-icon](https://user-images.githubusercontent.com/8650435/79673626-6a113900-81ec-11ea-9d43-bed4ca7e8ffc.png)
![location-icon](https://user-images.githubusercontent.com/8650435/79673627-6aa9cf80-81ec-11ea-8b17-98a06731702a.png)